pkg/kv/export/utils.go

```diff
@@ -0,0 +1,71 @@
+package export
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"errors"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/treeverse/lakefs/pkg/logging"
+)
+
+var ErrIllegalFilePath = errors.New("illegal file path")
+
+// Untar takes a destination dir and a reader to a tar file.
+// Looping over the tar file, extracting files into dir
+func Untar(dir string, r io.Reader) error {
```

Comment:
Ignore if I wrote it before - but still think that this is too much at this point